.circleci/do-rtl-build.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# create the different verilator builds
+# argument is the make command string
+
+# turn echo on and error on earliest command
+set -ex
+
+# init all submodules
+cd $HOME/project
+./scripts/init-submodules-no-riscv-tools.sh
+
+# enter the verisim directory and build the specific config
+cd sims/verisim
+make clean
+
+# run the particular build command
+make JAVA_ARGS="-Xmx2G -Xss8M" $@
```

Comment:
Is this actually used? `JAVA_ARGS` seems like a boom-template only thing.

Reply:
This is used to make sure that elaboration doesn't blow over the 2G limit that CircleCI has. Ill add it to the makefile.